src/app/components/MatchCardDog/MatchCardDog.module.css

```
@@ -0,0 +1,29 @@
+.container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 1em;
+  background-color: var(--secondary-background-color);
+  padding: 1em;
+  border-radius: 1em;
+}
+
+.matchImage {
```

You could be less specific here (and other classes), because it's the module file of your `Match`.

```suggestion
.image {
```